README.md

```diff
@@ -12,6 +12,9 @@ This role installs Kong on Debian systems, configured to use Postgres for storag
 
 For more information on Kong please visit [getkong](https://getkong.org/).
 
+If you want to test locally using Ubuntu Xenial, add `VAGRANT_BOX=ubuntu/xenial64`
+to `.make` file.
+
```

Comment:
I have reviewed and tested on my laptop locally. Apart from the small doc change it looks good to me.